gradle/common.gradle

```diff
@@ -19,7 +19,7 @@ repositories {
 }
 
 ext {
-    groovyVersion = '2.4.8'
+    groovyVersion = '2.4.12'
```

Comment:
I suggest to remove this setting here too and inline it completely into `build.gradle` to prevent the current confusion.